lib/dfe/analytics/initialisation_events.rb

```diff
@@ -2,46 +2,48 @@
 
 module DfE
   module Analytics
-    # DfE Analytics initialisation event
+    # DfE Analytics initialisation events
     # - Event should only be sent once, but NOT on startup as this causes errors on some services
     # - Event contains the dfe analytics version, config and other items
-    class Initialise
+    class InitialisationEvents
       # Disable rubocop class variable warnings for class - class variable required to control sending of event
       # rubocop:disable Style:ClassVars
-      @@initialise_event_sent = false # rubocop:disable Style:ClassVars
+      @@initialisation_events_sent = false # rubocop:disable Style:ClassVars
 
-      def self.trigger_initialise_event
-        new.send_initialise_event
+      def self.trigger_initialisation_events
+        new.send_initialisation_events
       end
 
-      def self.initialise_event_sent?
-        @@initialise_event_sent
+      def self.initialisation_events_sent?
+        @@initialisation_events_sent
       end
 
-      def self.initialise_event_sent=(value)
-        @@initialise_event_sent = value # rubocop:disable Style:ClassVars
+      def self.initialisation_events_sent=(value)
+        @@initialisation_events_sent = value # rubocop:disable Style:ClassVars
       end
 
-      def send_initialise_event
+      def send_initialisation_events
         return unless DfE::Analytics.enabled?
 
-        initialise_event = DfE::Analytics::Event.new
+        initialise_analytics_event = DfE::Analytics::Event.new
                                                 .with_type('initialise_analytics')
-                                                .with_data(initialisation_data)
+                                                .with_data(initialise_analytics_data)
                                                 .as_json
 
         if DfE::Analytics.async?
-          DfE::Analytics::SendEvents.perform_later([initialise_event])
+          DfE::Analytics::SendEvents.perform_later([initialise_analytics_event])
         else
-          DfE::Analytics::SendEvents.perform_now([initialise_event])
+          DfE::Analytics::SendEvents.perform_now([initialise_analytics_event])
         end
 
-        @@initialise_event_sent = true # rubocop:disable Style:ClassVars
+        DfE::Analytics::EntityTableCheckJob.perform_later
+
+        @@initialisation_events_sent = true # rubocop:disable Style:ClassVars
       end
 
       private
 
-      def initialisation_data
+      def initialise_analytics_data
         {
           analytics_version: DfE::Analytics::VERSION,
           config: {
```

Comment:
It might be useful to send the `entity_table_checks_enabled` config item here in the initialise event. I'm not sure though if Analytics dataform will use it though. What do you think @stevenleggdfe ?

Reply:
Yes please @asatwal @ericaporter ! This would enable us to switch checks on/off in Dataform depending